README.md

```diff
@@ -26,31 +26,51 @@ Next follow these instructions.
     PASSWORD: osp
     HOST: localhost
     ``` 
-    You may run the following commands for local setup of DB:
+    You may run the following commands for local setup of DB in Linux:
 
     ```
+    cd open-source-programs-backend
     sudo -i -u postgres
     createuser osp --pwprompt
     psql
     CREATE DATABASE osp;
     \c osp;
     GRANT ALL PRIVILEGES ON DATABASE osp to osp;
     ```
+2. you need to download API key file from your user-settings on Zulip. The file you download is named as 'download' or rename that to 'download'.
+3. Put that download file in the project's directory.
 
-
-2. To start the server:
+4. Move into the project's directory.
 
     ```
     cd open-source-programs-backend
+    ````
+5. You don't need to create virtual environment while working in Linux.
+    `````
     virtualenv venv
     source venv/bin/activate
     pip install -r requirements.txt
+    ````
+6. To run the migrations run: 
+   ````
```

Comment:
these type of code block markdown usually is 3 "`" instead of 4. Can you change the "````" to "```" please :)